### ISSUE TITLE
Can't build Libra/Wylt

### ISSUE THREAD

**Issue by glacambre** [kori/wylt#7]
Hi, thanks for building this project :).

On Ubuntu 18.04 x86_64 and with go 1.10.4, I can't compile Wylt/Libra:
```
>go build main.go
# command-line-arguments
./main.go:57:26: undefined: m
./main.go:102:28: undefined: logroot
./main.go:104:3: undefined: fmt
./main.go:109:9: assignment mismatch: 2 variables but 1 values
./main.go:111:3: undefined: fmt
./main.go:132:20: undefined: w
./main.go:138:15: undefined: playingNow
./main.go:139:11: undefined: current
./main.go:143:41: undefined: current
./main.go:149:34: undefined: current
./main.go:149:34: too many errors
```

I went back in the project's history and the last commit I can compile is e06a06e515ddcfac65a76498e70a7c38229085a8.

**Comment by kori** [comment]
Hey!

I realized I said this in other issues, but now I really do have time to
look at wylt again.

Right now I'm busy, but I'm gonna clone the project later today and look at
all the issues.

It's a promise! I even set a reminder!

**Comment by kori** [comment]
I'm working on this right now. We should get a working build today.

Apologies for the breakage.

**Comment by kori** [comment]
Solved in edde782
